pages/volunteer_role.tsx

```diff
@@ -172,8 +172,8 @@ const VolunteerRolePage = () => {
         {roleData.preferredQualifications ? (
           <RoleDescriptionSubSection
             title={'Preferred Qualifications'}
-            content={roleData.preferredQualifications}
-            list={false}
+            content={roleData.preferredQualifications.split(/\r?\n/g)}
```

Comment:
For consistency, we should move the parsing to VolunteerRoleService file. Weston put in a getSplitFieldRecord method there.

You will also have to check for empty/null lines.